swap_meet/vendor.py

```diff
@@ -0,0 +1,93 @@
+############### TEST WAVE 1 (5 tests) PASSED ###############
+############### TEST WAVE 2 (3 tests) PASSED ###############
+############### TEST WAVE 4 (3 tests) PASSED ###############
+############### TEST WAVE 6 (6 tests) ###############
+# from item.swap_meet import Item - why is this not needed?
+
+### test 1.1 PASSED ### test 1.2 PASSED ###
+class Vendor:
+    def __init__(self, inventory = None): #get_by_category,
+    # you can't make an argument/parameter of a mutable data type
+    # In python, if the inventory default is set to an empty list as the paramenter - then one instance update ...will change all the instances -
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+        # self.get_by_category = get_by_category() -- this is a Vendor method and does not need to be an attribute.
+
+### test 1.3 PASSED ###
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+### test 1.4 PASSED ### test 1.5 PASSED ###
+    def remove(self, item):
+        if self.inventory == []:
+            return None
+        elif item not in self.inventory:
+            return False
```

Comment:
This could be merged

```suggestion
        if item not in self.inventory:
            return False
```